src/open_inwoner/templates/pages/profile/edit.html

```diff
@@ -1,45 +1,64 @@
 {% extends 'master.html' %}
-{% load i18n l10n form_tags %}
+{% load i18n l10n grid_tags form_tags anchor_menu_tags icon_tags button_tags %}
+
+{% block sidebar_content %}
+    <div class="profile-nav">
+        {% icon icon="west" icon_position="before" outlined=True %}
+        <p class="profile-nav__content">Terug naar {% button href="profile:detail" text=_("mijn gegevens") transparent=True %}</p>
+    </div>
+{% endblock sidebar_content %}
 
 {% block extra_head %}
-    {{form.media}}
+    {{ form.media }}
 {% endblock %}
 
 {% block content %}
-    <h1 class="h1" id="title">
-        {% trans "Profiel bewerken" %}
-    </h1>
-
-    {% render_form id="profile-edit" method="POST" form=form form_action=form_action enctype="multipart/form-data" %}
-        {% csrf_token %}
-
-        {% with request.user as user %}
-            {% if user.is_digid_user_with_brp %}
-                {% input form.display_name %}
-                {% input form.email %}
-                {% input form.phonenumber %}
-                {% if user.contact_type == "begeleider" %}
-                    {% input form.image no_help=True %}
-                    {% image_crop form.cropping %}
-                {% endif %}
-            {% else %}
-                {% input form.first_name %}
-                {% input form.infix %}
-                {% input form.last_name %}
-                {% input form.display_name %}
-                {% input form.email %}
-                {% input form.phonenumber %}
-                {% input form.street %}
-                {% input form.housenumber %}
-                {% input form.postcode %}
-                {% input form.city %}
-                {% if user.contact_type == "begeleider" %}
-                    {% input form.image no_help=True %}
-                    {% image_crop form.cropping %}
-                {% endif %}
-            {% endif %}
-        {% endwith %}
-        {% form_actions primary_text=_("Opslaan") primary_icon="arrow_forward" secondary_href='profile:detail' secondary_text=_('Terug') secondary_icon='arrow_backward' secondary_icon_position="before" enctype="multipart/form-data" %}
-    {% endrender_form %}
+
+    <div class="profile__grid">
```

Comment:
This doesn't seem to render properly on mobile devices http://127.0.0.1:8000/mijn-profiel/edit/

![Screenshot from 2024-02-20 10-42-43](https://github.com/maykinmedia/open-inwoner/assets/29249171/aecdbadd-7444-4b36-90c3-af9a1a82f155)

Is that an oversight or is this out of scope for this PR?

Reply:
hmm could be that I didn't have `npm run watch` running, looks good now :+1: 

![image](https://github.com/maykinmedia/open-inwoner/assets/29249171/7e455d88-a35b-4556-9760-ff3cee5e86a0)